downstream/modules/platform/ref-platform-non-inst-database-inventory.adoc

```diff
@@ -2,11 +2,12 @@
 
 [id="ref-standlone-platform-inventory_{context}"]
 
-= Example {PlatformName} inventory file with an external managed database
+= Example inventory file for a database on the automation controller node or a non-installer managed database
```

Comment:
Super long title that sounds like it covers most scenarios. How about shortening to something like "Database inventory file example"?

Reply:
It is not a database inventory file example. It is a inventory file to install AAP. I think that the proposed title is correct  because it follow the structure of the other sections.